api-common-java/src/main/java/com/google/api/core/ApiFutures.java

```diff
@@ -45,18 +45,30 @@
 public final class ApiFutures {
   private ApiFutures() {}
 
-  /*
-   * @deprecated Use {@linkplain #addCallback(ApiFuture, ApiFutureCallback, Executor) the
-   * overload that requires an executor}. For identical behavior, pass {@link
-   * com.google.common.util.concurrent.MoreExecutors#directExecutor}, but consider whether
-   * another executor would be safer.
+  /**
+   * @deprecated Use {@linkplain #addCallback(ApiFuture, ApiFutureCallback, Executor) the overload
+   *     that requires an executor}. For identical behavior, pass {@link
+   *     com.google.common.util.concurrent.MoreExecutors#directExecutor}, but consider whether
+   *     another executor would be safer.
    */
   @Deprecated
   public static <V> void addCallback(
       final ApiFuture<V> future, final ApiFutureCallback<? super V> callback) {
```

Comment:
One major intent of this ticket is to improve the reference API documentation found at https://cloud.google.com/java/docs/reference/api-common/latest/com.google.api.core.ApiFutures, so please make sure that each of these methods have entries for `@param`, `@return`, and `@throws` as appropriate.

Reply:
done.